therapy/schemas.py

```diff
@@ -159,6 +159,7 @@ class SourcePriority(IntEnum):
     HEMONC = 3
     DRUGBANK = 4
     CHEMIDPLUS = 5
+    CHEMBL = 6
```

Comment:
have we gotten confirmation from Alex that this is the correct placement for ChEMBL?

Reply:
This was added pretty arbitrarily to just give it a spot. I can ask Alex and as something potentially related, I think someone else had mentioned in the DGIdb meeting about weird licensing issues for ChEMBL that could affect using it?

Reply:
Back when we only had NCIt, ChEMBL, DrugBank, and Wikidata, ChEMBL was 2 (right after NCIt).

Reply:
I got confirmation from Alex to put this after DrugBank. The new order is: 

`RXNORM = 1
    NCIT = 2
    HEMONC = 3
    DRUGBANK = 4
    CHEMBL = 5
    CHEMIDPLUS = 6
    WIKIDATA = 10`